### ISSUE TITLE
HEREDOC syntax in `translation_script` -  `morpheus_rest_option_list`

### ISSUE THREAD

**Issue by mbjorgan** [gomorpheus/terraform-provider-morpheus#128]
After importing a rest option list it comes in as a single string and lines divided by `\r\n`. 
If I change this to HEREDOC syntax the apply is ok, but it will always show change on the next plan/apply

**Comment by martezr** [comment]
I'll take a look at getting this fixed. You're likely already aware but something that I often use is referencing a file instead of using the inline string. I find this easier to read and update especially with more complex javascript code or even other resource types.

```
resource "morpheus_rest_option_list" "tf_example_rest_option_list" {
  name               = "tf_example_rest_option_list"
  description        = "Terraform REST option list example"
  visibility         = "private"
  source_url         = "https://api.github.com/repos/hashicorp/consul/releases"
  real_time          = true
  ignore_ssl_errors  = true
  source_method      = "GET"
  translation_script = file("${path.module}/translation.js")
  source_headers {
    name  = "Accept"
    value = "application/json"
  }
}
```

translation.js
```
for(var x=0;x < 5; x++) {
    results.push({name: data[x].name,value:data[x].name});
}
```

**Comment by mbjorgan** [comment]
Yeah, using a file gives the same update in-place as using inline string

**Comment by martezr** [comment]
Additional code changes have been made in the #133 pull request that adds logic to trim the payload before comparison that fixed the issue with the HEREDOC examples that I was using.

**Comment by martezr** [comment]
This issue has been addressed with the v0.9.3 release of the Terraform provider. Please re-open the issue if there are still issues with the Terraform plan.
